models/attachments.js

```diff
@@ -1,4 +1,4 @@
-Attachments = new FS.Collection('attachments', {
+Attachments = new FS.Collection('attachments', { // eslint-disable-line meteor/collections
```

Comment:
Example of a valid case where something other than `new Meteor.Collection()` is assigned to a collection.